harbor-cluster/templates/cluster-configuration.yaml

```diff
@@ -0,0 +1,21 @@
+ {{- if .Values.core.oidc.enabled }}
```

Comment:
Can we have several HarborConfigurations per Harbor instance ? If not, then this "if" condition should be lower in the code.

Reply:
yes it is possible to have several HarborConfigurations per Harbor instance